msbuild-example/README.md

```diff
@@ -148,30 +148,52 @@ Up until now, we saw how we can configure our project to resolve and deploy gene
 From the Artifactory UI, create the following repositories. You can read more about Nuget repositories in the [Nuget Repositories User Guide](https://www.jfrog.com/confluence/display/RTF/NuGet+Repositories)<br>
 * Local NuGet repository named *nuget-local*. 
 * Remote NuGet repository named *nuget-remote*.
-* Virtual NuGet repository named *nuget-virtual*. 
-Next, include *nuget-remote* and *nuget-local* in *nuget-virtual*.
+* Virtual NuGet repository named *nuget*. 
+Next, include *nuget-remote* and *nuget-local* in *nuget*.
 
 ### Build the Solution Using JFrog CLI
-CD into the solution's root directory and run the following command 
+CD into the solution's root directory and run the following command to set your project with the virtual repository (named *nuget*) you created.
+If you'd like JFrog CLI to use the **nuget** client, run:
 ```console
-> jfrog rt nuget restore nuget-virtual --build-name=nuget-build --build-number=1
+> jfrog rt nugetc
 ```
-The command restored the two project's dependencies by downloading them from Artifactory and also stored the build-info locally.
+and if you'd like it to use the **dotnety core CLI**, run:
+```console
+> jfrog rt dotnetc
+```
+The above commands created a directory named *.jfrog* at the root directory of the project.
+
+Now run one of the following commands, depending on whether you you'd like to use the **nuget** or **dotnet core CLI** clients.
```

Comment:
_...depending on whether **you you'd** like to use the nuget or **dotnet core CLI** clients._ -> _... depending on whether **you'd** like to use the **nuget** or **.NET core CLI** clients._